roles/k3s_server/defaults/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+server_init_args: >-
+  {% if ansible_hostname == hostvars[groups['master'][0]]['ansible_hostname'] %}
+    --cluster-init
+  {% else %}
+    --server https://{{ apiserver_endpoint }}:6443
```

Comment:
Should the port be variablized? Everything else seems variablized.

Reply:
Done